sourcery_analytics/cli/data.py

```diff
@@ -19,7 +22,8 @@ def from_dict(
         cls,
         d: typing.Dict[str, typing.Any],
         threshold_settings: ThresholdSettings,
-    ):
+    ) -> "ThresholdBreach":
```

Comment:
**suggestion (docstrings):** Please update the docstring for function: `from_dict`

Reason for update: Clarification of the construction process from a dictionary.

Suggested new docstring:
```
"""Constructs a ThresholdBreach instance from a dictionary.

"""
```